terracotta/config.py

```diff
@@ -73,6 +73,12 @@ class TerracottaSettings(NamedTuple):
     #: MySQL database password (if not given in driver path)
     MYSQL_PASSWORD: Optional[str] = None
 
+    #: PostgreSQL database username (if not given in driver path)
+    POSTGRESQL_USER: Optional[str] = None
+
+    #: PostgreSQL database password (if not given in driver path)
+    POSTGRESQL_PASSWORD: Optional[str] = None
```

Comment:
We might want to rename these to `SQL_USER`/`SQL_PASSWORD` and then deprecate the `MYSQL_` fields?

Reply:
@chapmanjacobd I have thought about this, but I think it would be best to avoid using the `PG*` variables. Every other env var that Terracotta reads is prefixed with `TC_` so for consistency I think we should stick to that.

Reply:
@nickeopti Okay, I won't add them here then. If we rename the existing variables, it will break backwards compatibility for users who are already using MySQL. Since we are heading for a 1.0 release, maybe that's fine?

Reply:
I'll leave decisions regarding backwards compatibility and deprecation behaviour to @dionhaefner, @j08lue and/or you @mrpgraae. I don't know the use cases well enough

Reply:
See #258